Sources/MarkdownView/Renderer/Renderer+List.swift

```diff
@@ -45,17 +46,17 @@ extension Renderer {
             let depth = unorderedList.listDepth
             let handler = interactiveEditHandler
             let rawText = text
+            let configuration = configuration
             VStack(alignment: .leading, spacing: configuration.componentSpacing) {
                 ForEach(itemContent.indices, id: \.self) { index in
                     let listItem = listItems[index]
                     HStack(alignment: .firstTextBaseline) {
                         if listItem.checkbox != nil {
                             CheckboxView(listItem: listItem, text: rawText, handler: handler)
                         } else {
-                            SwiftUI.Text("•")
-                                .font(.title2)
-                                .fontWeight(.black)
-                                .padding(.leading, depth == 0 ? 12 : 0)
+                            SwiftUI.Text(configuration.unorderedListBullet)
+                                .font(configuration.unorderedListBulletFont)
```

Comment:
Font is no need to be customized here. I've tested title2 is the best font to fit the content. Just give the flexibility to customize list bullet is okay.